tests/cucumber/unit.tags

```diff
@@ -16,7 +16,9 @@
 @unit.responses.231
 @unit.responses.unlimited_assets
 @unit.sourcemap
+@unit.stateproof.paths
```

Comment:
Will a follow up address the other stateproof tags:  `@unit.stateproof.responses` and `` @unit.stateproof.responses.msgp` ? Or is that not slated to be tested in JS?

Reply:
Added in recent commits.

Reply:
Decided to remove these--I got them to work in browserless mode, but the firefox tests end up failing.

I did a bunch of digging into this and could not end up getting msgp decoding working for them in the returned responses within the firefox environment.

Reply:
@jasonpaulos - I believe you have a deeper understanding of the browser-based tests so I wanted to ping you into the discussion as well.

Reply:
Resolved in afbd042